lntest/itest/lnd_forward_interceptor_test.go

```diff
@@ -224,6 +225,45 @@ func testForwardInterceptor(net *lntest.NetworkHarness, t *harnessTest) {
 	// that tests the payment final status for the held payment.
 	cancelInterceptor()
 	wg.Wait()
+
+	// Verify that we don't get notified about already completed HTLCs
```

Comment:
Test addition is solid, only a few style nits here to ensure lines are wrapped to 80 columns (line 262, etc). 

Confirmed that w/o the prior commit, this test fails as those intercepted packets are being replayed.